[2025-05-02T17:29:44.780+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-05-02T17:29:44.809+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data_pipeline.preprocess_data manual__2025-05-02T17:29:31.509192+00:00 [queued]>
[2025-05-02T17:29:44.821+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data_pipeline.preprocess_data manual__2025-05-02T17:29:31.509192+00:00 [queued]>
[2025-05-02T17:29:44.828+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2025-05-02T17:29:44.850+0000] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): preprocess_data> on 2025-05-02 17:29:31.509192+00:00
[2025-05-02T17:29:44.866+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=635) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-05-02T17:29:44.871+0000] {standard_task_runner.py:63} INFO - Started process 637 to run task
[2025-05-02T17:29:44.868+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'weather_data_pipeline', 'preprocess_data', 'manual__2025-05-02T17:29:31.509192+00:00', '--job-id', '72', '--raw', '--subdir', 'DAGS_FOLDER/weather_pipeline_dag.py', '--cfg-path', '/tmp/tmplmkddoe1']
[2025-05-02T17:29:44.873+0000] {standard_task_runner.py:91} INFO - Job 72: Subtask preprocess_data
[2025-05-02T17:29:44.935+0000] {task_command.py:426} INFO - Running <TaskInstance: weather_data_pipeline.preprocess_data manual__2025-05-02T17:29:31.509192+00:00 [running]> on host 0b0b6d30f653
[2025-05-02T17:29:45.012+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_data_pipeline' AIRFLOW_CTX_TASK_ID='preprocess_data' AIRFLOW_CTX_EXECUTION_DATE='2025-05-02T17:29:31.509192+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-02T17:29:31.509192+00:00'
[2025-05-02T17:29:45.014+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-05-02T17:29:45.026+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-05-02T17:29:45.028+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python /opt/***/scripts/preprocess_data.py']
[2025-05-02T17:29:45.048+0000] {subprocess.py:86} INFO - Output:
[2025-05-02T17:29:48.312+0000] {subprocess.py:93} INFO - âœ… Preprocessing complete.
[2025-05-02T17:29:49.364+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-05-02T17:29:49.378+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-05-02T17:29:49.497+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=weather_data_pipeline, task_id=preprocess_data, execution_date=20250502T172931, start_date=20250502T172944, end_date=20250502T172949
[2025-05-02T17:29:49.572+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-05-02T17:29:49.600+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-02T17:29:49.604+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
